docs/log-levels.md

```diff
@@ -48,13 +49,13 @@ Some libraries and situations may not be entirely clear with regards to what log
 `trace` level logging:
 
 - Could include various additional information about a request, such as various diagnostics about created data structures, the state of caches or similar, which are created in order to serve a request.
-- Could include "begin operation" and "end operation" logs.
-  - However, please consider using [swift-distributed-tracing](https://github.com/apple/swift-distributed-tracing) to instrument "begin" and "end" events, as tracing can be more efficient than logging. Logging may need to create string representations of the durations, log levels, and other fields.
+- Could include "begin operation" and "end operation" logging statements.
```

Comment:
May be worth describing what an "operation" is in this context -- is it a method, or a business operation / unit of work?

Reply:
Also, is there a way to get [dump](https://developer.apple.com/documentation/swift/1539127-dump) output into logging?

Reply:
> Also, is there a way to get dump output into logging?

not really; that's not all that much used to be honest. If anything people can use `String(reflecting:)` which is the "debug description" rather than `String(describing:)` (`"\()"`) which is the normal description of an object.

Reply:
> May be worth describing what an "operation" is in this context -- is it a method, or a business operation / unit of work?


Ok I'll add that -- it can be any of those really; most often a function though yes